identity-service/sequelize/migrations/20200706140854-download-email.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+const models = require('../../src/models')
+
+/**
+ * Adds boolean columns 'hasSentDownloadAppEmail' and 'hasSignedInNativeMobile'
+ * to UserEvents to track if a user has signed in on native mobile and if not
+ * if they have been sent an email to download the app.
+ * Migration includes inserting row for each user so that they are not sent an
+ * email moving forward.
+ */
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addColumn('UserEvents', 'hasSentDownloadAppEmail', {
+        type: Sequelize.BOOLEAN,
+        allowNull: false,
+        defaultValue: false
+      }, { transaction })
+      await queryInterface.addColumn('UserEvents', 'hasSignedInNativeMobile', {
+        type: Sequelize.BOOLEAN,
+        allowNull: false,
+        defaultValue: false
+      }, { transaction })
+
+      const users = await models.User.findAll({
```

Comment:
how big is this on prod? will it fit in memory? probs should be ok?

Reply:
There are ~40k rows in the users table, it's only selecting the walletAddress column which is a varchar(255) so that's 40k * 255 / 1e6 = ~10.2 MB which should be fine.